lib/active_model/serializer/adapter.rb

```diff
@@ -1,12 +1,14 @@
 module ActiveModel
   class Serializer
     class Adapter
+      UnknownAdapterError = Class.new(ArgumentError)
```

Comment:
I think `class UnknownAdapterError < ArgumentError; end` is a more common syntax.